CHANGELOG.md

```diff
@@ -836,9 +835,40 @@ Non-backwards compatible changes
   Therefore, only _declarations_ of `PartialOrder` records will need their field names
   updating.
 
-* NB (issues #1214 #2098) the corresponding situation regarding the `flip`ped
-  relation symbols `_≥_`, `_>_` (and their negated versions!) has not (yet)
-  been addressed.
+* NB (issue #2098) the corresponding situation regarding the `flip`ped
+  relation symbols `_≥_`, `_>_` has not (yet) been addressed.
+
+### (Issue #1214) Reorganisation of the introduction of negated relation symbols under `Relation.Binary`
+
+* Previously, negated relation symbols `_≰_` (for `Poset`) and `_≮_` (`TotalOrder`)
+  were introduced in the corresponding `Relation.Binary.Properties` modules, for re-export.
```

Comment:
Why are we not mentioning `_≮_` here?

Reply:
Do you mean `\<~n`? Because it was never used in the library before #2099 introduced it as the symbol for `Preorder`. But I can do so if you think it adds additional force?

Reply:
Just to check: I was confused by your original question, because I do mention `_≮_` here... so I (perhaps mistakenly) presumed you had meant to type an their symbol. Could you restate the question please, so that I can better see the drift of it? Sorry to be obtuse!